rts/System/Sound/OpenAL/Sound.cpp

```diff
@@ -454,9 +454,17 @@ void CSound::OpenLoopbackDevice(const std::string& deviceName)
 {
 	assert(curDevice == nullptr);
 
-	hasAlcSoftLoopBack = (alcIsExtensionPresent(nullptr, "ALC_SOFT_loopback") == AL_FALSE);
-	if (!hasAlcSoftLoopBack)
+#ifdef ALC_SOFT_loopback
+	LOG("[Sound::%s] ALC_SOFT_loopback define was set!", __func__);
+#endif
+
+	hasAlcSoftLoopBack = (alcIsExtensionPresent(nullptr, "ALC_SOFT_loopback") == AL_TRUE);
+	if (!hasAlcSoftLoopBack) {
+		LOG("[Sound::%s] ALC_SOFT_loopback extension NOT found using alcIsExtensionPresent(...)!", __func__);
 		return;
+	} else {
+		LOG("[Sound::%s] ALC_SOFT_loopback extension found using alcIsExtensionPresent(...)!", __func__);
```

Comment:
Ditto for above, we're more interested in logging the cases where compatibility is poorer, in this case using sdl audio logging might not be necessary.